### PR TITLE
Migrate from Travis to Github Actions

### PR DESCRIPTION
**Pull request by greg0ire**

### CODE REVIEW

File: .github/workflows/continuous-integration.yml [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    name: "PHPUnit with SQLite"
+    runs-on: "ubuntu-20.04"
+
+    strategy:
+      matrix:
+        php-version:
+          - "7.1"
+          - "7.2"
+          - "7.3"
+          - "7.4"
```
Comment (alexpott) [inline .github/workflows/continuous-integration.yml:30]: Add PHP 8.0?

**Review by SenseException — APPROVED** [review]

**Comment by greg0ire** [comment]
For some reason I can't get the static analysis check to run…

**Comment by greg0ire** [comment]
That was because of a single missing quote… :facepalm:

**Comment by SenseException** [comment]
It's probably time to sleep. :sleepy:

**Comment by greg0ire** [comment]
Wise words… one last PR and I go to bed, I promise!

**Pull request merged**